bindings/cpp/Cargo.toml

```diff
@@ -25,7 +25,7 @@ homepage = "https://opendal.apache.org/"
 license = "Apache-2.0"
 repository = "https://github.com/apache/opendal"
 rust-version = "1.75"
-version = "0.44.3"
+version = "0.45.3"
```

Comment:
Should be `0.44.4`?

Reply:
No. The CMakeList version is wrongly 0.45.x and we should not trace back. Hopefully we can keep align from this version.

Reply:
Got it, thanks!